install/kubernetes/cilium/templates/cilium-agent/clusterrole.yaml

```
@@ -41,6 +41,13 @@ rules:
   - get
   - list
   - watch
+- apiGroups:
+  - ""
+  resources:
+  - events
+  verbs:
+  - create
+  - patch
```

Maybe only have theses when the drop emitter is enabled?
```suggestion
{{- if and .Values.hubble.enabled .Values.hubble.dropEventEmitter.enabled }}
- apiGroups:
  - ""
  resources:
  - events
  verbs:
  - create
  - patch
{{- end }}
```

---

Thanks